src/edu/hillel/lesson23/builder/SportCarBuilder.java

```diff
@@ -0,0 +1,35 @@
+package edu.hillel.lesson23.builder;
+
+import lombok.Getter;
+
+@Getter
+public class SportCarBuilder implements Builder {
```

Comment:
Usually, Builder is creating as inner class
It's not a mistake but a best practices